packages/react-components/react-datepicker-compat/src/components/DatePicker/useDatePicker.tsx

```diff
@@ -353,18 +353,29 @@ export const useDatePicker_unstable = (props: DatePickerProps, ref: React.Ref<HT
     : 'outline';
 
   const [triggerWrapperRef, popupRef] = usePopupPositioning(props);
-  const root = slot.always(restOfProps, {
```

Comment:
You are missing a change file, you need to run this in root:
```sh
yarn change
```

Reply:
@sopranopillow I have made this change :)